server/src/main/java/org/elasticsearch/action/search/OpenPointInTimeRequest.java

```diff
@@ -76,6 +81,9 @@ public void writeTo(StreamOutput out) throws IOException {
         if (out.getTransportVersion().onOrAfter(TransportVersions.V_8_12_0)) {
             out.writeOptionalWriteable(indexFilter);
         }
+        if (out.getTransportVersion().onOrAfter(TransportVersions.ALLOW_PARTIAL_SEARCH_RESULTS_IN_PIT)) {
+            out.writeBoolean(allowPartialSearchResults);
+        }
```

Comment:
I am not sure when this is needed since this request is generally handled by the node that receives the rest request but I updated the code to throw if that happens.